SDK/AppCenterPush/Microsoft.AppCenter.Push/Microsoft.AppCenter.Push.csproj

```
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

We should revert and remove `xmlns` as we did in https://github.com/microsoft/appcenter-sdk-dotnet/pull/962/files#diff-6faca249d7490bb37b4cd2c6d5a79c90R2. Otherwise, cake script won't work.

```suggestion
<Project Sdk="Microsoft.NET.Sdk">
```

---

Not true, I already verified cake build